content/applications/marketing/marketing_automation/campaign_templates/double_optin.rst

```diff
@@ -0,0 +1,153 @@
+
+.. _marketing_automation/template/using-double-optin:
+
+Use the Double Opt-in campaign template
+=======================================
+
+Open the :menuselection:`Marketing Automation` app, and select the :guilabel:`Double Opt-in`
+campaign template to create a new campaign for confirming consent.
+
+.. tip::
+   The campaign templates do not display by default when there are existing *Marketing Automation*
+   campaigns. To display the campaign templates, type the name of a campaign that does not exist in
+   the database into the :guilabel:`Search...` bar, then press :kbd:`Enter`.
+
+   For example, searching for `empty` displays the campaign template cards again, as long as there
+   is not a campaign with the name "empty" in the database.
+
+Campaign configuration
+----------------------
+
+Upon creation of the campaign, the campaign form loads with a new preconfigured campaign.
+
+The :guilabel:`Target` and :guilabel:`Filter` configuration of the campaign is as follows:
+
+- :guilabel:`Name`: `Double Opt-in`
+- :guilabel:`Responsible`: The user who created the campaign.
+- :guilabel:`Target`: :guilabel:`Mailing Contact`
+- :guilabel:`Unicity based on`: :guilabel:`Email (Mailing Contact)`
+- :guilabel:`Filter`:
+
+  - :guilabel:`Email` :guilabel:`is set`
+  - :guilabel:`Blacklist` :guilabel:`is not` :guilabel:`set`
+  - :guilabel:`Mailing lists` :guilabel:`contains` `Newsletter`
+
+.. important::
+   The :guilabel:`Target` model of the campaign should not be modified. Changing the
+   :guilabel:`Target` model with activities in the :guilabel:`Workflow` invalidates the existing
+   activities in the :guilabel:`Workflow`.
+
+   The *Double Opt-in* campaign template is intended to only use the :guilabel:`Mailing Contact`
+   model.
+
+The campaign loads two activities in the :guilabel:`Workflow` section of the campaign: an email
+activity with a child server action activity that triggers *on click*.
+
+By default, the `Confirmation` email activity is set to trigger :guilabel:`1 Hours` after the
+beginning of the workflow. In other words, the email is sent 1 hour after a new contact is added to
+the *Newsletter* mailing list.
+
+The email activity uses the preconfigured *Confirmation* email template, which contains a button for
+the contact to click to confirm their consent.
+
+To modify the email template, select the :icon:`fa-envelope-o` :guilabel:`Templates` smart button at
+the top of the campaign form. Then, in the list of templates, select the `Confirmation` email
+template.
+
+Be sure to personalize the contents of the email template; however, it is recommended to keep the
+contents of double opt-in confirmation emails short and to-the-point.
+
+The default confirmation button links directly to the database's website homepage. Click on the
+button to edit the button text and URL.
+
+.. tip::
+   To provide a streamlined user experience for the contact, consider :doc:`creating a page on the
```

Comment:
```suggestion
   To provide a streamlined experience for the contact, consider :doc:`creating a page on the
```
I've been instructed by ZST in recent PRs to avoid using the term "user" -- hence, my suggestion here